src/DebugSystem.cpp

```diff
@@ -88,369 +187,247 @@ DebugSystem::DebugSystem(uint32_t rom_start, uint32_t rom_size, uint32_t ram_sta
     }
 }
 
+
+DebugSystem::DebugSystem(void) :
```

Comment:
avoid duplicate code by using delegating constructor:

```
DebugSystem::DebugSystem() : DebugSystem(-1, 0, -1, 0) {}
```

Reply:
FYI: what i meant is to completely remove the content of the function body, because it would do exactly the same as the other constructor. this code here executes the other constructor.

see: https://github.com/tum-ei-eda/etiss/pull/37/commits/c43174f23cb3848fe36c29c04130fccb098ebec9